pkg/client/receipts/client.go

```diff
@@ -0,0 +1,93 @@
+package receipts
```

Comment:
Feels like this package should live in `pkg/service/egresstracking` since it's only used there.